charts/hivemq-operator/values.yaml

```diff
@@ -187,6 +187,12 @@ hivemq:
 
         [[ ! -f drivers/postgres-jdbc.jar ]] &&
         curl -L https://jdbc.postgresql.org/download/postgresql-42.2.14.jar --output drivers/jdbc/postgres.jar
+    - name: hivemq-google-cloud-pubsub-extension
```

Comment:
Can we move the pub/sub extension to just after the kafka extension? That way we can group available enterprise extensions by topic. Once a kinesis becomes available it would move to after pub/sub.